demos/python_demos/3d_segmentation_demo/3d_segmentation_demo.py

```diff
@@ -153,21 +171,35 @@ def resample_np(data, output_shape, order):
     return interpolation.zoom(data, zoom=factor, order=order)
 
 
-def read_image(test_data_path, series_name, sizes=(128, 128, 128)):
+def read_image(test_data_path, data_name, sizes=(128, 128, 128), is_series=True):
     images_list = []
     handle = None
     bboxes = np.zeros(shape=(len(DATA_SUFFIXES),) + (2, 3))
 
-    for j, s in enumerate(DATA_SUFFIXES):
-        image_handle = read_nii_header(test_data_path, series_name, s, separate_folder=False)
-        handle = image_handle
-        image = image_handle.get_data().astype(np.float32)
+    if is_series:
+        for j, s in enumerate(DATA_SUFFIXES):
+            image_handle = read_nii_header(test_data_path, data_name, s, separate_folder=False)
+            handle = image_handle
+            image = image_handle.get_data().astype(np.float32)
 
-        mask = image > 0.
-        bboxes[j] = bbox3(mask)
-        image = normalize(image, mask)
+            mask = image > 0.
+            bboxes[j] = bbox3(mask)
+            image = normalize(image, mask)
 
-        images_list.append(image.reshape((1, 1,) + image.shape))
+            images_list.append(image.reshape((1, 1,) + image.shape))
+            original_shape = image.shape
+    else:
+        image_handle = read_nii_header(test_data_path, data_name, separate_folder=False)
+        handle = image_handle
+        image = image_handle.get_data().astype(np.float32)
+        # Reading order is specified for data from http://medicaldecathlon.com/
+        for j in (1, 3, 0, 2):
+            img = image[:, :, :, j]
```

Comment:
I think it would be better to call _this_ variable `image`, since it directly corresponds to the `image` variable from the other branch. The variable you called `image` should be called something else.